controllers/admin/AdminCategoriesController.php

```diff
@@ -606,7 +625,6 @@ public function renderForm()
 
         $this->tpl_form_vars['shared_category'] = Validate::isLoadedObject($obj) && $obj->hasMultishopEntries();
         $this->tpl_form_vars['PS_ALLOW_ACCENTED_CHARS_URL'] = (int)Configuration::get('PS_ALLOW_ACCENTED_CHARS_URL');
-        $this->tpl_form_vars['displayBackOfficeCategory'] = Hook::exec('displayBackOfficeCategory');
```

Comment:
Three modules are hooked on this thing (18642-ilmminmaxorderqtypro	19015-productvideovimeoyoutube	19487-ilmqtyindozen), I guess the hook should be left here since it makes sense to want to add additional tools to the category page.